js/main.js

```diff
@@ -47,6 +51,8 @@ function getUrlAndOpenMainWindow() {
         let title = 'Error loading configuration';
         electron.dialog.showErrorBox(title, title + ': ' + err);            
     });
+
+    applyContextMenu();
```

Comment:
might be diff but for now the same.  also want to move code into here since this involves creating windows.